python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
-# interval is `R`, which confidence interval is [708.3,780.0] (MPa).
-# This is why this parameter seems the most important in this case.
+# interval is `R`, which confidence interval is [741, 758] (MPa).
+# This is why this parameter seems to be the most important in this case.
 
 # %%
+# We now plot the predicted output stress depending on the input
+# strain before and after calibration.
+# sphinx_gallery_thumbnail_number = 3
 graph = calibrationResult.drawObservationsVsInputs()
 graph.setLegendPosition("topleft")
 view = viewer.View(graph)
 
 # %%
-# We see that there is a good fit after calibration, since the predictions after calibration (i.e. the green crosses) are close to the observations (i.e. the blue crosses).
+# We see that there is a good fit after calibration, since the predictions
+# after calibration are close to the observations.
 
 # %%
+# We can also plot the predicted stress depending on the observed stress.
 graph = calibrationResult.drawObservationsVsPredictions()
 view = viewer.View(graph)
 
 # %%
-# We see that there is a much better fit after calibration, since the predictions are close to the diagonal of the graphics.
+# We see that there is a much better fit after calibration, since the
+# predictions are close to the diagonal of the graphics.
 
 # %%
+# The :meth:`~openturns.CalibrationResult.getObservationsError` method
+# returns the estimated distribution of the observation error.
+# This is necessarily a Gaussian distribution, because this is the distribution
+# that we assume when we use least squares.
+# By hypothesis, the distribution has a zero mean (this is a property
+# of linear least squares).
+# The standard deviation is estimated from the residuals after calibration.
 observationError = calibrationResult.getObservationsError()
 observationError
 
 # %%
+# In order to validate that the distribution of the residuals is
+# Gaussian after calibration, we use the
+# :meth:`~openturns.CalibrationResult.drawResiduals` method.
 graph = calibrationResult.drawResiduals()
-graph.setLegendPosition("topleft")
-view = viewer.View(graph)
+view = viewer.View(
+    graph,
+    figure_kw={"figsize": (6.0, 4.0)},
+    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+)
+plt.subplots_adjust(right=0.6)
 
 # %%
-# The analysis of the residuals shows that the distribution is centered on zero and symmetric.
+# The analysis of the distribution of the residuals after calibration
+# shows that the distribution is centered on zero and is symmetric.
 # This indicates that the calibration performed well.
+# With the least squares method, the standard deviation of the observation
+# noise is estimated from the data.
+# On the plot, we see that the Gaussian with zero mean and estimated
+# standard deviation is close to the distribution of the residuals
+# after calibration.
+# This shows that the distribution of the residuals is close to being Gaussian.
+
+# %%
+# Check that the results are Gaussian, using a Normal-plot.
```

Comment:
```suggestion
# This can also be seen on a Normal-plot.
```
I think this links the paragraph better to the previous one.

Reply:
Excellent. Fixed.